### PR TITLE
union instead of concatenation, because duplicate fields

### PR DESCRIPTION
**Pull request by wyattisimo**

### CODE REVIEW

**Comment by nateberkopec** [comment]
Oh hi Jared!

Do I get to bother you to add a test case?

**Comment by wyattisimo** [comment]
yes

sigh

**Comment by nateberkopec** [comment]
I like #295. I think it would be best if sanitize_fields was just a simple array. By default, the array is filled with the default list, but you can manipulate it as you would an array to modify.

**Comment by wyattisimo** [comment]
@nateberkopec added test case

File: spec/raven/configuration_spec.rb [hunk 1]
```diff
@@ -130,4 +130,16 @@
     end
   end
 
+  context 'configuration for sanitize fields' do
+    it 'should union default sanitize fields with user-defined sanitize fields' do
+      fields = Raven::Processor::SanitizeData::DEFAULT_FIELDS | %w(test monkeybutt)
```
Comment (nateberkopec) [inline spec/raven/configuration_spec.rb]: Shouldn't we use a field like "password" or something? Neither of the fields you're adding here are in DEFAULT_FIELDS

Comment (wyattisimo) [inline spec/raven/configuration_spec.rb]: Right, but I'm adding those to the list of default fields and assigning all of that to `sanitize_fields`.

Comment (nateberkopec) [inline spec/raven/configuration_spec.rb]: Right you are!

**Pull request merged**